src/marginalhist.jl

```diff
@@ -60,7 +60,7 @@
         left_margin --> 0mm
         bins := edges2
         y := y
-        xlims --> ylims
+        ylims --> ylims
```

Comment:
I'm not sure if this is the expected behaviour when `orientation := :h` though. But indeed ylims changes the lims of the vertical axis.